components/symbl_ai/actions/put-speaker-events/put-speaker-events.mjs

```diff
@@ -0,0 +1,35 @@
+import symblAIApp from "../../symbl_ai.app.mjs";
+
+export default {
+  key: "symbl_ai-put-speaker-events",
+  name: "Update Speaker Events",
+  description: "Update the Speaker Events of the Conversation. See the doc [here](https://docs.symbl.ai/docs/conversation-api/speaker-events/).",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    symblAIApp,
+    conversationId: {
+      propDefinition: [
+        symblAIApp,
+        "conversationId",
+      ],
+    },
+    speakerEvents: {
```

Comment:
There is some endpoint to list all users? It would be great if we could use `async options` here.

Reply:
Unfortunately not at this time.

Reply:
I see, ok